internal/testutil/acc/hcl.go

```diff
@@ -0,0 +1,52 @@
+package acc
```

Comment:
what's this hcl? sorry if I am missing something

Reply:
[HCL is the HashiCorp configuration language.](https://github.com/hashicorp/hcl)

Reply:
yes, but why we have a file named like that? even the method names themselves are not quite clear on what they are doing - suggest renaming both method names and the file name

Reply:
maybe `terraform_config.go` is better?
And renaming:
`acc.HclMap` -> `acc.TfConfigVariableMap`
`acc.HclLifecycleIgnore` - >`acc.TfConfigLifecycleIgnore`
WDYT?

Reply:
it looks like `config_formatter.go`
Better to use `formatToHCLMap`

Reply:
Renamed in 17981e8ed017543d8c287cc1bc2bfbc73404b9ed